### DISCUSSION TITLE
How to make the password reveal eye button bigger ?

### DISCUSSION THREAD

**Discussion by LaurentInSeattle** [AvaloniaUI/Avalonia#17551] [Q&A] [answered]
![Screenshot 2024-11-18 083627](https://github.com/user-attachments/assets/2b875cde-fafc-4f9d-8339-638661f84f54)

Hi all, 

What is the simplest way to style the password text box so that the "eye" toggle button becomes bigger? 
It seems like its size is hardcoded as 12 pixels in the default style, correct? 

Thank you.

**Comment by maxkatz6** [comment, marked as answer]
You can copy-paste and modify this style, as an option
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/TextBox.xaml#L248-L265

**Reply by LaurentInSeattle** [reply]
Thanks, that worked. I was wondering if there was anything simpler...
